src/intervals/functions.jl

```diff
@@ -373,3 +373,13 @@ function nthroot(a::Interval{T}, n::Integer) where T
     b = nthroot(bigequiv(a), n)
     return convert(Interval{T}, b)
 end
+
+"""
+Calculate `x mod y` where `x` is an interval and `y` is a positive divisor.
+"""
+function mod(x::Interval, y::Real)
+    @assert y > 0 "modulo is currently implemented only for a positive divisor."
+    division = x / y
+    fl = floor(division)
+    fl.lo < fl.hi ? 0..y : y * (division - fl)
```

Comment:
```suggestion
    fl.lo < fl.hi ? Interval(zero(y), y) : y * (division - fl)
```

For internal use, the `Interval` constructor is better. The others (`interval`, `..`) are for end users and have some extra checks.